src/richie/apps/courses/templates/courses/cms/person_detail.html

```diff
@@ -39,6 +39,17 @@ <h1 class="person-detail__title">
     </div>
   </div>
 
+  {% if current_page.publisher_is_draft or not current_page|is_empty_placeholder:"organizations" %}
+  <div class="organization-glimpse-list">
+    <h2 class="organization-glimpse-list__title">{% trans "Organizations" %}</h2>
+    {% placeholder "organizations" or %}
+      <p class="organization-glimpse-list__item organization-glimpse-list__item--empty">
```

Comment:
@mbenadda Not sure what is the correct markup here. This is the item to display when the list is empty...

Reply:
There is no CSS rule for empty modifier on a glimpse it's something only related to its list container.

Inside `.organization-glimpse-list` in `src/frontend/scss/objects/_organization_glimpses.scss` you may add:
```
    &__empty {
      @include sv-flex(1, 0, auto);
      padding: 1rem;
      text-align: center;
    }
```

And change your empty item HTML to:

```
<p class="organization-glimpse-list__empty">
```

 However there is no such rules in `.organization-list` also, since we conditioned many list blocks so they do not appear when empty.

Reply:
@sveetch done